src/main/java/org/broadinstitute/hellbender/utils/test/testers/AbstractMarkDuplicatesTester.java

```diff
@@ -50,7 +53,7 @@ public File getMetricsFile() {
     }
 
     @Override
-    public String getTestedClassName() { return getProgram().getClass().getSimpleName(); }
+    public final String getTestedToolName() { return getProgram().getClass().getSimpleName(); }
```

Comment:
Why did you make this change in a few places?

Reply:
This classes were implementing `CommandLineProgramTest`, which is the `CommandLineProgramTester` implementation for GATK (extending also `GATKBaseTest`).

Because this classes are in the testing framework, and they are only tester classes, they do not inherit from `BaseTest` anymore. The method for returning the tool name is called `getTestedToolName` in `CommandLineProgramTester` and thus here it should be the one overridden.

Thus, this is one of the results of finally separating the gatk-testing framework (for re-use with downstream projects) and the test from GATK.